app/main.py

```diff
@@ -1,8 +1,32 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        (Person(name=person["name"], age=person["age"]))
+    print(Person.people)
+
+    for person in people:
+
+        if "husband" in person and person["husband"] is not None:
+            Person.people[person["name"]].husband =\
+                Person.people[person["husband"]]
+            person_list.append(Person.people[person["name"]])
+
+        elif "wife" in person and person["wife"] is not None:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+            person_list.append(Person.people[person["name"]])
+
+        else:
+            person_list.append(Person.people[person["name"]])
```

Comment:
why don't u add persons to the list using first for loop ?

Reply:
May be I`m not right, but "for" loop is above?